plugins/linux-apt/main.cpp

```diff
@@ -315,22 +316,23 @@ int main(int argc, char** argv)
             printf("Unknown option: %s\n", argv[1]);
             return 1;
         }
+#endif // DEBUG
```

Comment:
nit: in the future we could add these to a few unit tests, also thinking it would be a good place to add tests that download e2e (instruct apt to download package -> verify package download succeeds -> test cleanup uninstalls package & clears apt cache)

Reply:
agreed